docs/performance-budgets.md

```diff
@@ -0,0 +1,177 @@
+# Performance Budgets (budget.json)
+
+Use a performance budget to prevent site performance from regressing over time.
```

Comment:
in the context of lighthouse we might want a more specific overview explanation, maybe something like

```suggestion
Use a performance budget to assert a custom file size or count limit per resource type and thresholds for each performance metric.
```

? feel free to reword just something unique to budgets would be nice :)

Reply:
related: patrick should we mention interop w/ LHCI here?

Reply:
yeah a good chance to plug the budgets interop there too :)